go/common/version/version.go

```diff
@@ -95,7 +95,7 @@ var (
 	// the runtime.
 	//
 	// NOTE: This version must be synced with runtime/src/common/version.rs.
-	RuntimeHostProtocol = Version{Major: 3, Minor: 0, Patch: 0}
```

Comment:
I believe we already bumped this to `3` in master (21.1 has `2`), so another bump is not needed.